frontend/containers/discover-map/map-help/component.tsx

```diff
@@ -0,0 +1,207 @@
+import { FC, useState } from 'react';
+
+import { HelpCircle, Layers, MapPin, Search, ZoomIn, ZoomOut, X as XIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Modal from 'components/modal';
+import { theme } from 'tailwind.config';
+
+import clusterExemple from 'svgs/map/cluster-exemple.svg';
+
+import { MapHelpProps } from '.';
+
+export const MapHelp: FC<MapHelpProps> = () => {
+  const { formatMessage } = useIntl();
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="">
+      <Button
+        className="flex items-center justify-center w-8 h-8 px-1.5 py-1.5 text-gray-800 bg-white rounded shadow focus-visible:outline-green-dark"
+        theme="naked"
+        size="smallest"
+        icon={() => <HelpCircle className="mr-0" />}
+        onClick={() => setOpen(true)}
+        aria-label={formatMessage({ defaultMessage: 'Open map help', id: 'q/Z5g4' })}
+      />
+      <Modal
+        open={open}
+        onDismiss={() => setOpen(false)}
+        title={formatMessage({ defaultMessage: 'Map help', id: '5eVTzC' })}
+        theme="naked"
+      >
+        <div className="fixed z-50 w-full px-6 pt-6 bg-white sm:px-10">
+          <div className="flex justify-end">
+            <Button
+              theme="naked"
+              size="smallest"
+              onClick={() => setOpen(false)}
+              className="text-gray-400 focus:text-black hover:text-black group"
+              aria-label={formatMessage({ defaultMessage: 'Close map help', id: 'z7aNcb' })}
+            >
+              <span className="sr-only">
+                <FormattedMessage defaultMessage="Close" id="rbrahO" />
+              </span>
+              <Icon
+                icon={XIcon}
+                className="inline-block w-6 h-6 transition-all fill-current group-hover:rotate-180 group-focus-within:rotate-180"
+              />
+            </Button>
+          </div>
+          <h3 className="font-serif text-3xl font-semibold text-green-dark">
+            <FormattedMessage defaultMessage="Help" id="SENRqu" />
+          </h3>
+        </div>
+        <div className="px-6 mt-32 mb-24 sm:px-10 sm:mb-40">
```

Comment:
If not using the “naked” theme, you could simplify to:
```suggestion
        <div className="mt-6 mb-32">
```